lib/resources/r4/encounter_patch.yaml

```diff
@@ -0,0 +1,75 @@
+---
+name: Encounter
+field_name_base_url: http://hl7.org/fhir/R4/encounter-definitions.html#Encounter
+operations:
+  - name: replace-period-start
+    path: /period/start
+    operation: replace
+    cardinality: 0..1
+    type: dateTime
+    url: http://hl7.org/fhir/datatypes-definitions.html#Period.start
+    description: The start time of the encounter.
+    example: |
+      [
+        {
+          "op": "replace",
+          "path": "/period/start",
+          "value":  "2015-09-01T00:00:00.000Z"
+        }
+      ]
```

Comment:
Update the examples to just be the patch operation without the surrounding array.

Reply:
Updated. https://github.com/cerner/fhir.cerner.com/pull/240/commits/2f088fe9f198e6cb9d48c1be55d513f5eb8c9f72#diff-97d84edc87e609cdbf80d6ad20cbc681R13